commonConfiguration/.vscode/launch.json

```diff
@@ -0,0 +1,47 @@
+{
+  "version": "0.2.0",
+  "inputs": [
+    {
+      "id": "functionName",
+      "description": "Enter the name of the function to test",
+      "default": "health",
+      "type": "promptString"
+    }
+  ],
+  "configurations": [
+    {
+      "name": "Debug a lambda function λ",
+      "type": "node",
+      "request": "launch",
+      "cwd": "${workspaceFolder}",
+      "runtimeExecutable": "yarn",
+      "args": [
+        "serverless",
+        "invoke",
+        "local",
+        "-f",
+        "${input:functionName}",
+        "--path",
+        "functions/${input:functionName}/handler.mock.json"
+      ],
+      "sourceMaps": true,
+      "smartStep": true,
+      "outFiles": ["**/.esbuild/**/*.js"],
+      "protocol": "inspector",
+      "autoAttachChildProcesses": true,
+      "console": "integratedTerminal",
+      "outputCapture": "console"
+    },
```

Comment:
We can remove the inputs and the first configuration, since there are no lambdas in the repo

Reply:
This will be useful in the future, when we add the orchestrator backend

Reply:
fair enough